pphtml.py

```diff
@@ -230,7 +230,7 @@ def scanImages(self):
         # verify image files are jpg (reported as JPEG) or png
         for fd in self.filedata:
             t = fd.split("|")
-            if t[1] != "JPEG" and t[1] != "PNG":
+            if t[1] != "JPEG" and t[1] != "PNG" and t[1] != "SVG":
```

Comment:
Suggestion: a bit more pythonic:
```suggestion
            if t[1] not in ("JPEG", "PNG", "SVG"):
```

Reply:
Done.